src/redux/books/books.js

```diff
@@ -0,0 +1,35 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
```

Comment:
- [ ] Kindly ensure that the value of this action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux), which should look like:      
```javascript
 const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';